tests/src/python/test_qgsserver_accesscontrol.py

```diff
@@ -39,6 +39,7 @@
     'xsi:schemaLocation="http://www.opengis.net/wfs http://schemas.opengis.net/wfs/1.0.0/WFS-capabilities.xsd" '
     'xmlns:gml="http://www.opengis.net/gml" '
     'xmlns:ows="http://www.opengis.net/ows" '
+    'xmlns:qgs="http://qgis.org" '
```

Comment:
is this another one we should tolerantly patch in?

Reply:
I'm not sure this is needed, given that we inject all the other namespaces in this XMLNS variable. Clients are supposed to post a valid XML to the server. There's no particular reason to inject the qgs namespace rather than any other namespaces, which we don't do (cf https://github.com/qgis/QGIS/blob/master/src/server/services/wfs/qgswfstransaction_1_0_0.cpp#L71), IMHO.  Maybe @elpaso has a clue about that ?

Reply:
Gentle ping @elpaso

Reply:
> TBH I don't understand the question.

The question is if we should add a hack in the server production code to add the "qgs" namespace to the input provided by the client in case it would be missing, or if we can just fix the test case as I did